### PR TITLE
Add sonar scanner to test build step

### PR DESCRIPTION
**Pull request by gitizenme**
Changes
- Add sonar scanner to test build, update readme

### CODE REVIEW

**Comment by gubbin** [comment]
Your review environment is available at http://review-hello-cammis-pr34.dev.cammis-test.com

File: sonar-project.properties [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# must be unique in a given SonarQube instance
+sonar.projectKey=CA-MMIS:HELLO
```
Comment (jeffmaher) [inline sonar-project.properties]: Super small nitpick:

```suggestion
sonar.projectKey=CA-MMIS:HELLO_CAMMIS
```

Comment (gitizenme) [reply sonar-project.properties]: Done!

**Comment by gitizenme** [comment]
@jeffmaher Implemented the change you suggested. SonarCube is now a build check on a PR (see check list above).

File: .circleci/config.yml [hunk 2]
```diff
@@ -31,7 +34,7 @@ jobs:
           command: npm install
       - run:
           name: Update container repositories
-          command: sudo apt-get update
+          command: apk update
```
Comment (gubbin) [inline .circleci/config.yml]: I'd just leave this out, unless absolutely necessary.

**Pull request merged**